pydm/widgets/image.py

```diff
@@ -310,19 +331,22 @@ def redrawImage(self):
             img = self.image_waveform
 
         if len(img) <= 0:
+            self.redraw_timer.start()
             return
         if self._normalize_data:
             mini = self.image_waveform.min()
             maxi = self.image_waveform.max()
         else:
             mini = self.cm_min
             maxi = self.cm_max
+        img = self.process_image(img)
```

Comment:
I think the image processing has to occur before `if self._normalize_data...`, because processing can change minimum and maximum values of the image.

Reply:
I agree!